<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>insertDepartment</class>
 <widget class="QDialog" name="insertDepartment">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>542</width>
    <height>437</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>insertDepartment</string>
  </property>
  <widget class="QLabel" name="Title">
   <property name="geometry">
    <rect>
     <x>132</x>
     <y>52</y>
     <width>335</width>
     <height>55</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>335</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>400</width>
     <height>300</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:36pt;&quot;&gt;添加部门&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="AddDepartment">
   <property name="geometry">
    <rect>
     <x>169</x>
     <y>331</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>添加</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Cancel">
   <property name="geometry">
    <rect>
     <x>296</x>
     <y>331</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="departmentNum">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>120</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>请输入部门编号</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Name">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>120</y>
     <width>36</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>编号：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Password">
   <property name="geometry">
    <rect>
     <x>133</x>
     <y>171</y>
     <width>36</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>名称：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="departmentName">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>170</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
   <property name="placeholderText">
    <string>请输入部门名称</string>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_TEL">
   <property name="geometry">
    <rect>
     <x>133</x>
     <y>218</y>
     <width>60</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>部门职能：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="departmentFunction">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>220</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>请输入部门职能</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Education">
   <property name="geometry">
    <rect>
     <x>133</x>
     <y>274</y>
     <width>72</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>负责人编号：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="departmentPrincipal">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>270</y>
     <width>138</width>
     <height>20</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>请输入负责人编号</string>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
